docs/api-docs/analytics/bodl-overview.md

```diff
@@ -0,0 +1,406 @@
+# Big Open Data Layer (BODL) - Open Beta
+
+This functionality is currently in beta. Share your feedback with us using the [Partner Portal](https://partners.bigcommerce.com/). 
+
+## Overview
+
+The Big Open Data Layer (shortened as `BODL`, pronounced 'Bottle') is a JavaScript object that exposes storefront data points to BigCommerce and third-party analytics integrations. Instantiating a standard `BODL` object on a BigCommerce-hosted or headless storefront can make the site faster and more efficient by allowing BODL to fetch the data all at once instead of each script independently fetching the same data.
+
+BigCommerce checks your storefront for a `BODL` instance once per page render. To ensure that the analytics providers you've chosen have the complete and correct set of data points they require, initialize a standard `BODL` instance with the following schema. If you want to make an alternate custom version of this data layer object with a unique schema, name it something else.
+
+<!-- theme: info -->
+> When you create custom `BODL` instances, we recommend choosing clear, unique names that are easy to identify programmatically: for example, `BODL_YOUR_APP_NAME`.
+
+### Standard BODL schema with Stencil object values
+
+Consult [Stencil object reference](/theme-objects) for more about object properties.
+
+| BODL Property or Method | Stencil Object or Parameter |
```

Comment:
@traci707 can you check back through all the variables created in the instantiation script to verify that this is all of them?

Reply:
@slsriehl I just finished verifying all the variables.  I was able to verify: breadcrumbs, productId, productTitle, productCurrency, productPrice, products, wishlist, brand, search, categoryName, categoryProducts, order, cartItemAdded, getCartItemContentId and getQueryParamValue